ckanext/switzerland/tests/test_format_utils.py

```diff
@@ -75,25 +59,25 @@ def test_prepare_resource_format(self):
             'format': 'xml'
         }
         resource_with_valid_format_cleaned = ogdch_format_utils.prepare_resource_format(
-            resource_with_valid_format.copy(), format_mapping=ogdch_mixin.format_mapping)
+            resource_with_valid_format.copy())
         self.assertEquals('XML', resource_with_valid_format_cleaned['format'])
 
         resource_with_invalid_media_type_but_valid_after_slash = {
             'download_url': 'http://download.url',
-            'media_type': 'cat/gif',
-            'format': 'gif'
+            'media_type': 'rdf/xml',
+            'format': 'xml'
```

Comment:
It seems like you've changed the behaviour of the resource format mapping here - was that deliberate? In the old code, there is this block at the end of the function `prepare_resource_format`:

```
    # if format could not be mapped and media_type exists use this value  # noqa
    if (not resource.get('format') and resource.get('media_type')):
        resource['format'] = resource['media_type'].split('/')[-1]
```
I think this is where the format of this resource would be mapped to `gif`. This behaviour should be preserved after the refactoring.

Reply:
But GIF is not a valid format in the format mapping list. So I did not really understand that test. Why should it be mapped to gif? Could you please eplain this too me, why that test case is valid? I also compared with opendata.swiss, there is no dataset, that is mapped to 'gif'.